roles/ovirt-host-deploy/meta/main.yml

```diff
@@ -20,3 +20,4 @@ galaxy_info:
 dependencies:
   - ovirt-host-deploy-facts
   - ovirt-host-deploy-firewalld
+  - ovirt-provider-ovn-driver
```

Comment:
Please move this role up. I think firewalld should be the last role in the list to ensure that all services are available for firewalld

Reply:
Done.